raiden/api/v1/encoding.py

```diff
@@ -327,3 +327,41 @@ class ConnectionsLeaveSchema(BaseSchema):
     class Meta:
         strict = True
         decoding_class = dict
+
+
+class EventTransferSentFailedSchema(BaseSchema):
+    block_number = fields.Integer()
+    identifier = fields.Integer(missing=None)
+    event = fields.Str()
+    reason = fields.Str()
+
+    class Meta:
+        fields = ('block_number', 'event', 'reason')
+        strict = True
+        decoding_class = dict
+
+
+class EventTransferSentSuccessSchema(BaseSchema):
+    block_number = fields.Integer()
+    identifier = fields.Integer(missing=None)
+    event = fields.Str()
+    amount = fields.Integer(required=True)
```

Comment:
I believe this is not required, since we are using the strict mode, same thing for the other requires